platform/macos/export/export_plugin.cpp

```diff
@@ -458,6 +458,7 @@ void EditorExportPlatformMacOS::get_export_options(List<ExportOption> *r_options
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "application/additional_plist_content", PROPERTY_HINT_MULTILINE_TEXT), ""));
 
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "xcode/platform_build"), "14C18"));
+	// TODO(sgc): Need to set appropriate version when using Metal
```

Comment:
Reminder

Reply:
I think it is actually the `application/min_macos_version` that will need to be set if Metal is the chosen driver.

Reply:
All `xcode` properties are there only because App Store require them to be set (it should reflect Xcode/SDK that export template was build with). The `application/min_macos_version` is indeed the property used to set supported version in the plist, but again it's prob ably only be a warning and have fallback to MoltenVK by default.